catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -14,16 +14,19 @@
 -->
 <metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
 
-    <OCD name="CSW Federated Source" id="Csw_Federated_Source" description="CSW Federated Source">
+    <OCD name="CSW Specification Profile Federated Source" id="Csw_Federated_Source"
```

Comment:
This was mostly me trying to provide more information as to when one should be used over the other.  The `Specification Profile` is meant to be the generic implementation that should interoperate with any CSW service that adheres to the CSW 2.0.2 specification.  I also added a simple configuration for the DDF CSW profile.